pkg/apis/kops/validation/aws.go

```diff
@@ -49,3 +53,15 @@ func awsValidateAdditionalSecurityGroups(fieldPath *field.Path, groups []string)
 
 	return allErrs
 }
+
+func awsValidateMachineType(fieldPath *field.Path, machineType string) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if machineType != "" {
```

Comment:
This is great! But can we only run this when the cluster is on aws? I think this will bust GCE and vsphere ;)